src/analytics.rs

```diff
@@ -9,15 +9,24 @@ use breez_sdk_core::{
     InvoicePaidDetails, Payment, PaymentDetails, PaymentFailedData, ReceivePaymentResponse,
 };
 use log::{error, info, warn, Level};
+use num_enum::TryFromPrimitive;
 use parrot::{AnalyticsClient, AnalyticsEvent, PayFailureReason, PaymentSource};
 use std::sync::{Arc, Mutex};
 use std::time::SystemTime;
 use uuid::Uuid;
 
+#[derive(Debug, PartialEq, TryFromPrimitive, Clone)]
+#[repr(u8)]
+pub enum AnalyticsConfig {
+    Enabled,
+    Disabled,
+}
+
```

Comment:
If this can ever only be `Enabled` or `Disabled`, why not just use a Boolean instead? Are you expecting more sophisticated configurations in the future?

Reply:
I opted for the enum approach because it leaves less room for confusion when compared to a bool and also because it allows for more configurations if we ever need them.